torch2jax/__init__.py

```diff
@@ -415,6 +421,23 @@ def flatten(input, start_dim=0, end_dim=-1):
   return jnp.reshape(_v(input), input.shape[:start_dim] + (-1,))
 
 
+@implements(torch.max, out_kwarg=True, Torchish_member=True)
+def max(input, *args):
```

Comment:
Discussion: 
how can we handle the function overloading? the current implementation works okay if all args are pos args. But would fail if we call `max(a, dim=1)` or `max(a, other=torch.tensor(...))`.

Reply:
Done